apps/prairielearn/src/pages/legacyQuestionFile/legacyQuestionFile.ts

```diff
@@ -29,17 +28,16 @@ router.get(
 
     const questionIds = await chunks.getTemplateQuestionIds(question);
 
-    /** @type {chunks.Chunk[]} */
     const templateQuestionChunks = questionIds.map((id) => ({
-      type: 'question',
+      type: 'question' as const,
       questionId: id,
     }));
-    const chunksToLoad = /** @type {chunks.Chunk[]} */ ([
+    const chunksToLoad: chunks.Chunk[] = [
       {
-        type: /** @type {const} */ ('question'),
-        questionId: question.id,
+        type: 'question' as const,
+        questionId: question.id as string,
```

Comment:
Is the casting actually needed here?

Reply:
It is, unfortunately.

Reply:
Hmm, I don't see any type errors after removing it?

Reply:
But no type errors, yeah.

Reply:
IMO the right solution here is going to be figuring out how to get types on the contents of `res.locals` one way or another (or eliminate `res.locals` completely?). I'd consider casting `any as string` to be lying to the type checker 🙃 there's a time and place for that, but I don't think this is one of them.

Reply:
I think typing `res.locals` is for another PR.

Reply:
Completely agreed; please don't try to do that here!